internal/highway/crud_registry.go

```diff
@@ -1,57 +1,122 @@
 package highway
 
 import (
-	context "context"
-	"fmt"
+	"context"
 	"net/http"
 
 	"github.com/gin-gonic/gin"
 	rtv1 "github.com/sonr-io/sonr/x/registry/types"
 	rt "go.buf.build/grpc/go/sonr-io/blockchain/registry"
 )
 
+// ParamsHTTP
+// @Summary
+// @Schemes
+// @Description WhoIsHTTP
+// @Tags Registry
+// @Produce json
+// @Success      200  {string}  message
+// @Failure      500  {string}  message
+// @Router /registry/who_is[GET]
+func (s *HighwayServer) ParamsHTTP(c *gin.Context) {
+	params, err := s.cosmos.QueryParams()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	c.JSON(http.StatusOK, params)
+}
+
+// WhoIsHTTP
+// @Summary
+// @Schemes
+// @Description WhoIsHTTP
+// @Tags Registry
+// @Produce json
+// @Success      200  {string}  message
+// @Failure      500  {string}  message
+// @Router /registry/who_is[GET]
+func (s *HighwayServer) WhoIsHTTP(c *gin.Context) {
+	names, err := s.cosmos.QueryAllNames()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	c.JSON(http.StatusOK, names)
+}
+
+// WhoIsDIDHTTP
+// @Summary
+// @Schemes
+// @Description WhoIsDIDHTTP
+// @Tags Registry
+// @Produce json
+// @Success      200  {string}  message
+// @Failure      500  {string}  message
+// @Router /registry/who_is/:did [GET]
+func (s *HighwayServer) WhoIsDIDHTTP(c *gin.Context) {
+	if did := c.Param("did"); did != "" {
+		res, err := s.cosmos.QueryName(did)
+		if err != nil {
+			c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+			return
+		}
+
+		c.JSON(http.StatusOK, res)
+		return
+	}
+
+	c.JSON(http.StatusBadRequest, gin.H{"error": "did is required"})
+}
+
+// StartRegisterName starts the registration process for a Sonr Account.
 // @Summary Start Register Name
 // @Schemes
 // @Description StartRegisterName starts the registration process and returns a PublicKeyCredentialCreationOptions. Initiating the registration process for a Sonr Account.
 // @Tags Registry
 // @Produce json
 // @Success      200  {string}  message
 // @Failure      500  {string}  message
-// @Router /name/register/start/:username [get]
+// @Router /name/register/start/:username [GET]
 func (s *HighwayServer) StartRegisterName(c *gin.Context) {
 	if username := c.Param("username"); username != "" {
-		// Check if user exists and return error if it does
-		if exists := s.cosmos.NameExists(username); exists {
+		if s.cosmos.NameExists(username) {
 			c.JSON(http.StatusConflict, gin.H{"error": "username already exists"})
 		}
 
-		// Save Registration Session
-		options, err := s.webauthn.SaveRegistrationSession(c.Request, c.Writer, username, s.cosmos.AccountName())
+		options, err := s.webauthn.SaveRegistrationSession(
+			c.Request,
+			c.Writer,
+			username,
+			s.cosmos.AccountName(),
+		)
```

Comment:
remove formatting as this goes against the go fmt convention to my understanding

Reply:
Sorry, this is valid go formatting. ... duh, resolving this